frontend/pages/beta/model/[modelId].tsx

```diff
@@ -28,6 +29,7 @@ export default function Model() {
           tabs={[
             { title: 'Overview', path: 'overview', view: <Overview model={model} /> },
             { title: 'Releases', path: 'releases', view: <Releases model={model} /> },
+            { title: 'Access Requests', path: 'access-requests', view: <AccessRequests model={model} /> },
```

Comment:
Minor, but in for routing I've normally just gone with `access` rather than `access-request`. Feel free to leave as-is if you want, though.

Reply:
I'll align with what you've already done